Parameters.java

```diff
@@ -74,7 +74,9 @@ public class Parameters {
 	public static double sdStep = 0.3;
 	public static boolean mut2D = false;						// whether to mutate in a full 360 degree arc
 	public static boolean fixedStep = false;					// whether to fix mutation step size
-	public static String startingSequence = "AGTC";
+	public static String startingSequence = "AGTC";				// default starting sequence
+	public static String alphabetType = "nucleotides";          // default sequence to consist of nucleotides
```

Comment:
Enum might be better than a string here, no?

Reply:
Did you want to change this?

Reply:
`alphabetType` in `parameters.yml` uses a String, which is why I have this field in `Parameters.java` as a String. 

(This is similar to `phenotypeSpace`)